scrapper/tests/test_twitter.py

```diff
@@ -6,21 +6,29 @@ class twitterTest(testBase):
     def test_root(self):
         response = self.client.get('/twitter/')
         assert response.status_code == 400
-        #response_data = response.get_data(as_text = True)
-        #assert "You should really" in response_data
 
     def test_user_request(self):
         response = self.client.get('/twitter/flubbercwb')
+        assert response.status_code == 202
+
+    def test_user_found(self):
+        from scrapper.models.user import User
```

Comment:
I get the reason to import stuff on functions inside `manage.py`. But there's no excuse to do that on the tests. Move it all to the top of the file. And use `isort` command if possible.